indico/modules/events/papers/templates/display/call_for_papers.html

```diff
@@ -0,0 +1,75 @@
+{% from 'events/papers/display/_paper_list.html' import render_user_papers_list %}
+{% from 'events/reviews/_common.html' import render_timeline_section %}
+
+{% block title %}
+    {%- trans %}Call for Papers{% endtrans -%}
+{% endblock %}
+
+{% block content %}
+    <div class="call-for-papers">
+        {% set cfp = event.cfp %}
+        {% if cfp.start_dt %}
+            {{ render_timeline_section(cfp) }}
+        {% endif %}
+        {% if cfp.announcement %}
+            <section>
+                <div class="cfp-announcement">{{ cfp.announcement }}</div>
+            </section>
+        {% endif %}
+        <section class="cfp-infoline-section">
+            {{ render_cfp_infoline(event, cfp, show_icon=false) }}
+        </section>
+        {% if event.paper_templates %}
+            <section>
+                <div class="header">
+                    <div class="header-row">
+                        <h3>{% trans %}Paper templates{% endtrans %}</h3>
+                    </div>
+                </div>
+                <ul class="template-list">
+                    {% for tpl in event.paper_templates|sort(attribute='id')|sort(attribute='name') %}
+                        <li>
+                            <a href="{{ url_for('.download_template', tpl) }}" class="attachment {{ icon_from_mimetype(tpl.content_type, 'icon-file') }}">
+                                <span>{{ tpl.name }}</span>
+                            </a>
+                            {% if tpl.description %}
+                                <i class="info icon-info" title="{{ tpl.description }}"></i>
+                            {% endif %}
+                        </li>
+                    {% endfor %}
+                </ul>
+            </section>
+        {% endif %}
+        {% if contributions %}
+            <section>
+                <div class="header">
+                    <div class="header-row">
+                        <h3>{% trans %}My papers{% endtrans %}</h3>
+                        <div class="toolbar thin">
+                            <div class="group">
+                                <div id="filter-statistics" class="group">
+                                    {{- render_displayed_entries_fragment(contributions|length, contributions|length) -}}
+                                </div>
+                                <div class="group">
+                                    <span class="i-button label icon-search"></span>
+                                    <input type="text" id="search-input" placeholder="{% trans %}Enter #id or search string{% endtrans %}">
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                {{ render_user_papers_list(event, contributions) }}
+                <div id="filter-placeholder"></div>
+            </section>
+        {% endif %}
```

Comment:
We need to show a message if the user has no contributions (in case he gets to this page somehow through a direct link)